### PR TITLE
Feature/A1ODT-564-defaultBackendService-change

### PR DESCRIPTION
**Pull request by burakbengi**
- namespace as variable
- added extra values in alias
- disabled dB ingress
- testing on targetrevision (won't merge with feature name)

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Changes look good to me. We need to test that on an isolated cluster and then change the revisions to HEAD again

File: apps/argocd/overlays/core/environments/dev.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   project: default
   source:
     repoURL: https://github.com/catenax-ng/k8s-cluster-stack
-    targetRevision: HEAD
+    targetRevision: feature/A1ODT-564-defaultBackendService-change
```
Comment (carslen) [inline apps/argocd/overlays/core/environments/dev.yaml:12]: After PR has been approved and merged, the linked branch won't be available anymore (as of repositoriy setting/branch protection _delete branch after merge_). Same for `ingress-applicationset.yaml`.

**Review by carslen — CHANGES_REQUESTED** [review]

File: apps/argocd/overlays/core/environments/dev.yaml (hunk 1 above)
Comment (burakbengi) [reply apps/argocd/overlays/core/environments/dev.yaml:12]: Yes, exactly. We first want to create an example cluster for testing, where we can test this branch and than delete these lines. I am not resolving this conversation until that change :)

**Comment by carslen** [comment]
Implemented with PR #277 and #278

**Pull request closed without merging**